analyzer_test.go

```diff
@@ -194,6 +194,21 @@ func testAnalyzerBuilder(platformAPI string) func(t *testing.T, when spec.G, it
 					h.AssertEq(t, md.Metadata, platform.LayersMetadata{})
 				})
 			})
+
+			when("run image is provided", func() {
+				it.Before(func() {
+					analyzer.RunImage = image
```

Comment:
I was surprised to see the same `s0m3D1g3sT` value, but I guess using `image` as `RunImage` is convenient.